README.rst

```diff
@@ -1,113 +1,136 @@
 
 .. image:: https://travis-ci.org/yeti/yurt.svg?branch=master
     :target: https://travis-ci.org/yeti/yurt
 
 Dependencies
 ------------
-- Vagrant
-- Python 2.7
+- Docker for Mac/Windows
+- Python 2.7/3.x
 
 Setup
 -----
+
+Install the Pip package
+
 .. code-block:: shell
 
     pip install yak-yurt
-    vagrant plugin install vagrant-vbguest
+
+Alternatively, you can install from git
+
+.. code-block:: shell
+
+    git clone git@github.com:yeti/yurt.git
+    cd yurt
+    pip install --editable .
 
 Usage
 -----
 
 - Get HELP for a Yurt command
+
+.. code-block:: shell
+
+    yurt [COMMAND] --help
+
+- Create a new Django project
+
 .. code-block:: shell
 
-    yurt deploy --help
+   yurt new
 
+- Add remote environment context to Django project
+
+.. code-block:: shell
 
-- Create a Django project and a Vagrant VM instance
+    cd path/to/yurt-project
+    yurt env add
+
+- Import environment context from zip file
 
 .. code-block:: shell
 
-    cd new_proj
-    yurt new_project (--git_repo=<git repo link>) (--vault)
+    cd path/to/yurt-project
+    yurt env import path/to/env-context.zip
 
-- Adds a remote server target to the Django project
+- Export current environment context to zip file
 
 .. code-block:: shell
 
-    cd new_proj
-    yurt remote_server (--help)
+    cd path/to/yurt-project
+    yurt env export <context-name>
 
-- Deploys to a remote server target (must be inside the Django project git repo)
+- Run dev server
 
 .. code-block:: shell
 
-    cd new_proj/project_repo
-    yurt deploy
+    cd path/to/docker-compose.yml
+    docker-compose up -V
 
-- Setup a Yurt-started project (git ssh link required)
+- Stop dev server
 
 .. code-block:: shell
 
-    cd new_proj
-    yurt existing (--git_repo=<git repo link>)
+    cd path/to/docker-compose.yml
+    docker-compose down
 
-- Create a JSON file with Vault credentials (experimental)
+- Run remote server with `docker-compose`
 
 .. code-block:: shell
 
-    cd new_proj
-    yurt vault (--dest=<destination directory>)
+    cd path/to/docker-compose.yml
+    eval `docker-machine env <server-name>`
+    docker-compose -f docker-compose.<env>.yml up -d
 
 
 Notes on Project Structure
 --------------------------
-- After running either ``yurt existing`` or ``yurt new_project`` inside an empty directory ("new_proj") this is the structure:
+- Project will have the following structure upon running `yurt new`
 
 .. code-block:: shell
 
-    new_proj
-        |_ Vagrantfile
-        |_ project_repo
-            |_ manage.py
-            |_ requirements.txt
-            |_ config
-            |   |_ settings
-            |   |   |_ base.py
-            |   |   |_ local.py
-            |   |
-            |   |_ urls.py
-            |   |_ wsgi.py
+    <project_name> (PROJECT_ROOT)
+        |
+        |_ docker-compose.yml
+        |_ docker-compose.remote.yml
+        |
+        |_ django_app
+        |   |
+        |   |_ <project_name>
+        |   |   |_ settings.py
```

Comment:
Are we reverting back to `settings.py` since we're just using django's out the box new project? Typically we have base.py, dev.py, local.py, prod.py, etc.

Reply:
For 12-factor apps, it helps to move any variation between environments into environment variables (this makes it so that application code is mostly unaffected by environment changes).